core/src/main/java/jenkins/security/BasicApiTokenHelper.java

```diff
@@ -62,27 +64,4 @@
         
         return null;
     }
-    
-    //TODO could be simplified when the patch is merged in 2.129+
-    @SuppressWarnings("unchecked")
-    static boolean isLegacyTokenGeneratedOnUserCreation(){
```

Comment:
As this method was of scope "internal", I do not think somebody was using it. (code checked in Jenkins ecosystem and of course no other usage on the 15 August 2018)